docs/qs_first_boot.md

```diff
@@ -0,0 +1,101 @@
+# OVOS Quick Start - First Boot
+
+You now have a flashed boot medium with OVOS installed.  Now what?
+
+Insert your boot medium, and power on your device.
+
+**NOTE** On a RPi4 with a USB device, it is recommended to use the top-right USB3 port.
+
+Each image has a different experence when you first boot your device.
+
+[Buildroot - GUI](#Buildroot-GUI)
+
+[Raspbian-ovos - Headless](#Raspbian-ovos-Headless)
+
+**Notes on internet** While OVOS can run fully offline, with on device STT and TTS, your device will be lacking many of the things you expect from a smart assistant.  On a RPi, this includes the date and time because the Pi lacks a Real Time Clock, and therefor needs to connect to a server to set those on your device.
+
+## Buildroot - GUI
+
+When you first boot the Buildroot image, you will be greated with an OVOS splash screen as shown below.
+
+![](https://raw.githubusercontent.com/OpenVoiceOS/ovos_assets/master/Images/Screenshot%20-%20Buildroot%20bootsplash.png)
+
+As this is the first time you have booted your device, it might take a bit longer than normal as the system is preparing its local filesystem and extending it over the full size of the sdcard/USB device.
+Eventually the progress bar will be filled up indicating the Operating System has been fully booted, after which the ovos-shell animated loading screen will be shown.
```

Comment:
```suggestion
As this is the first time you have booted your device, it might take a bit longer than normal as the system is preparing its local filesystem and extending it over the full size of the sdcard/USB device.

Eventually the progress bar will be filled up indicating the Operating System has been fully booted, after which the ovos-shell animated loading screen will be shown.
```

More readable. Unless using pre-line indentations (which [line numbers / editor-based indentation] render mostly obsolete, and which Markdown disallows anyway) paragraph spaces should be used instead of just a line-break.

Then again, that is unless the preview deceives me, and GitHub now puts paragraph spacing between lines, like the suggestion preview demonstrates:

![image](https://github.com/OpenVoiceOS/community-docs/assets/42837531/184b830a-4dfd-4342-820a-1518a290bc2b)